docs/ADDRESS_LOOKUP.md

```diff
@@ -11,7 +11,63 @@
+          val shouldCompleteAddress = // whether an api call needs to be made or not
+          if (shouldCompleteAddress) {
+              val completeLookupAddress = // complete LookupAddress object
+              sendAddressLookupResult(AddressLookupDropInServiceResult.LookupComplete(completeLookupAddress))
+              return true
+          }
+          // If you do not need to use this callback either do not implement it, or return false.
+          return false
+      } 
+      ```
 - If you're integrating with standalone `CardComponent`:
     - Set `AddressLookupCallback` via `CardComponent.setAddressLookupCallback(AddressLookupCallback)` function to receive the related events.
     - Pass the result of these actions by calling `CardComponent.setAddressLookupResult(addressLookupResult: AddressLookupResult)`.
+    ```kotlin
+    cardComponent.setAddressLookupCallback(object : AddressLookupCallback {
+        override fun onQueryChanged(query: String) {
+            // Use the query parameter to make a call to your desired maps endpoint
+            // Map the address objects LookupAddress object and pass it back to Drop-in as follows:
+            val options: List<LookupAddress> = // mapped list of address objects
+            cardComponent.updateAddressLookupOptions(options)
+        }
+
+        override fun onLookupCompletion(lookupAddress: LookupAddress): Boolean {
+            // Optionally use this callback to get the complete details for a LookupAddress option selected by the shopper
+            // If you need to use this callback to make a call to get complete details for the selected address option you must return true.
+            // You can also use this function conditionally. In that case, return true if you perform an api call, false otherwise.
+            // Make a call to your desired maps endpoint and pass the complete LookupAddress object back to Drop-in as follows:
+            val shouldCompleteAddress = // whether an api call needs to be made or not
+                if (shouldCompleteAddress) {
+                    val completeLookupAddress = // complete LookupAddress object
+                    cardComponent.setAddressLookupResult(AddressLookupResult.Completed(completeLookupAddress))
+                    return true
+                }
+            // If you do not need to use this callback either do not implement it, or return false.
+            return false
+        }
+    })
+    ```
     - Delegate back pressed event to `CardComponent` by calling `CardComponent.handleBackPress()` which returns true if the back press is handled by Adyen SDK and false otherwise.
+    ```kotlin
+    override fun onBackPressed() {
+        if (cardComponent?.handleBackPress() == true) return
+        super.onBackPressed()
+    }
+    ```
+## Screenshots
+![address-lookup](https://github.com/Adyen/adyen-android/assets/6615094/ee671606-fbf4-4674-96ec-99a804e53d2e)
```

Comment:
Minor: it could be nice if all screenshots included the device frame as well. If you take the screenshot from inside Android Studio you can then [add a frame to it afterwards](https://developer.android.com/studio/debug/am-screenshot). Or even just drag and drop screenshots into this [device art generator](https://developer.android.com/distribute/marketing-tools/device-art-generator).

Reply:
I think we have the necessary information displayed through these images and that doesn't add much imo.